src/hooks/useVaultLocationAudit.ts

```diff
@@ -17,8 +17,7 @@ export default function useVaultLocationAudit(): UseVaultLocationAudit {
   const { selectedIds, data = [] } = useListContext<Item>()
   const record = useRecordContext<Item>()
   const { id } = useParams()
-  const ids = id ? [parseInt(id)] : selectedIds
-
+  const ids = id ? selectedIds : selectedIds
```

Comment:
This line doesn't look right.  Isn't it assigning `selectedIds` to `ids` whatever happens?